config/nginx.default

```diff
@@ -0,0 +1,22 @@
+# Default server configuration
```

Comment:
It would be nice if all docker-specific files could go into a docker/ subdirectory.